### DISCUSSION TITLE
password

### DISCUSSION THREAD

**Discussion by seth225** [open-webui/open-webui#3535] [General]
Forgot my password.   Is there a fix?

**Comment by Peter-De-Ath** [comment]
see https://docs.openwebui.com/troubleshooting#reset-admin-password

**Comment by chrleon** [comment]
Hmm, I can't get that to work. 

This is what I get back when I run the code for docker in a terminal
2024/07/31 10:34:44 socat[7] E execvp("bash", "bash", "-c", "apk", ...): No such file or directory

**Comment by ndjones** [comment]
Same issue, this looks relevant https://github.com/open-webui/open-webui/discussions/1566#discussioncomment-10281407
